src/Components/Navbar/style.ts

```diff
@@ -1,12 +1,60 @@
 import styled from "styled-components";
+import title from "../../assets/images/navbar-bg.png";
+import { Link } from "react-router-dom";
 
 export const NavStyle = styled.div`
-  background-color: black;
-  padding: 1.5rem;
+  nav {
+    background-image: url(${title});
+    background-repeat: no-repeat;
+    background-size: cover;
+    position: fixed;
+  }
 `;
 
-export const NavLink = styled.div`
-  color: dark black;
-  font-weight: 500;
-  margin-right: 1rem;
+export const StyledLink = styled(Link)`
+  color: #f00;
+  text-decoration: none;
+`;
+
+export const ImgStyle = styled.img`
+  height: 70px;
+
+  @media (max-width: 767px) {
+    height: 30px;
+  }
+
+  @media (min-width: 768px) and (max-width: 991px) {
+    height: 50px;
+  }
+
+  @media (min-width: 992px) and (max-width: 1199px) {
+    height: 60px;
+  }
+
+  @media (min-width: 1200px) and (max-width: 1399px) {
+    height: 90px;
+    width: 130px;
+  }
+
+  @media (min-width: 1400px) {
+    height: 85px;
+  }
+`;
+export const SocialMediaIconsStyle = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  a {
```

Comment:
why nested `a` tag?

Reply:
i used nested <a> tag for style social media icon